lib/rules/prefer-presence-queries.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+const { getDocsUrl, ALL_QUERIES_METHODS } = require('../utils');
+
+const QUERIES_REGEXP = new RegExp(
+  `^(get|query)(All)?(${ALL_QUERIES_METHODS.join('|')})$`
```

Comment:
what about `find`? https://testing-library.com/docs/dom-testing-library/api-queries#findby

Reply:
I guess it should be added but wasn't mentioned on original discussion.

Reply:
Thoughts about this: I don't think findBy should be included in this rule as it's the only async query so there is no alternative async query for it. When checking appearance you can use findBy and when checking disappearance you can use findBy too (wrapped with waitForElementToBeRemoved) so I think the rule is fine with only getBy and queryBy.

This said, I'll make sure I've used the appropriate language for describing the rule so it always refers to element presence/absence (sync, checked with getBy/queryBy) and never to element appearing/disappearing (async, checked with findBy).

Reply:
This makes sense, thanks 👍

Reply:
I hope I can update the doc accordingly and add the last check for var declarations tomorrow.